lua/lspconfig/stylelint_lsp.lua

```diff
@@ -0,0 +1,33 @@
+local configs = require('lspconfig/configs')
+local util = require('lspconfig/util')
+
+configs.stylelint_lsp = {
+  default_config = {
+    filetypes = {'javascript', 'javascriptreact', 'typescript', 'typescriptreact'};
+    root_dir = util.root_pattern('.stylelintrc', 'package.json');
+    settings = {};
+  },
+  docs = {
+    description = [[
+https://github.com/bmatcuk/stylelint-lsp
+
+Stylelint language server
+
+By default it doesn't have any executable file to run, so we have to make one ourselves.
+- `git clone https://github.com/bmatcuk/stylelint-lsp /path/to/project`
+- Make sure you have `yarn` installed: `npm i -g yarn`
+- `cd /path/to/project && yarn`
+- Create an executable file to run the server `echo "node /path/to/project/dist/index.js --stdio" > run-stylelint.sh`
+- `chmod +x run-stylelint.sh`
```

Comment:
Would be nice to fix this upstream since it's quite a hassle. From what I understand it should be fairly trivial, just add something like this to package.json:

```json
    "bin": {
        "stylelint-lsp": "index.js"
    }
```

Reply:
Yeah definitely a good idea, was thinking about this as well. But I had no idea how to build the binary globally using my local fork for testing purpose, I have never created any executable node library before. Searched on google but no luck 🙃. Any idea?

Reply:
Thanks, @dewey92 - I merged your PR into stylelint-lsp and cut a new release just now (v1.1.0)

Reply:
Super thanks @bmatcuk, will update my MR soon